src/libcharon/plugins/windows_dns/windows_dns_handler.c

```diff
@@ -212,7 +212,7 @@ static int GetTapAdapterInfo( IWbemServices *pSvc, int *iface_idx, wchar_t *dns1
 	{
 		// Allocate our strings
 		language = SysAllocString( L"WQL" );
-		query = SysAllocString( L"SELECT * FROM Win32_NetworkAdapterConfiguration where Description LIKE 'Sophos TAP Adapter'" );
+		query = SysAllocString( L"SELECT * FROM Win32_NetworkAdapterConfiguration where Description LIKE 'Sophos TAP Adapter%'" );
```

Comment:
Will there be any issue because of this expectation in line 226 that we assume only 1 matching result but this query returns more than 1?